src/script/conversation/MessageRepository.ts

```diff
@@ -824,6 +824,10 @@ export class MessageRepository {
       await this.conversationRepositoryProvider().makeSureMLS1to1ConversationIsEstablished(conversation);
     }
 
+    if (isMLS) {
+      void this.conversationRepositoryProvider().refreshMLSConversationVerificationState(conversation);
```

Comment:
Refresh conversation state before a message is sent in a MLS conversation.

Reply:
I think we should await that call actually (so that message sending is blocked in case we degrade)

Reply:
What do you mean by blocked? The refresh method does not throw any errors... Or you mean to have a degraded system message appearing first before the actual message?

Reply:
Discussed on a different chat, addressed the comment with https://github.com/wireapp/wire-webapp/pull/17056/commits/37b6517a93f70312aa080195ea42dc2cc2740130